tests/core/pyspec/eth2spec/test/altair/transition/test_transition.py

```diff
@@ -242,3 +244,164 @@ def test_transition_only_blocks_post_fork(state, fork_epoch, spec, post_spec, pr
 
     yield "blocks", blocks
     yield "post", state
+
+
+@fork_transition_test(PHASE0, ALTAIR, fork_epoch=3)
+def test_transition_with_finality(state, fork_epoch, spec, post_spec, pre_tag, post_tag):
+    """
+    Transition from the initial ``state`` to the epoch after the ``fork_epoch``,
+    including attestations so as to produce finality through the fork boundary.
+    """
```

Comment:
👍 on having docstring for each test case (which we probably should write more...).